components/omega/doc/devGuide/Testing.md

```diff
@@ -0,0 +1,322 @@
+(omega-dev-testing)=
+
+# Testing Code
+
+For instructions on how to build Omega and run CTest unit tests, see
+{ref}`omega-dev-quick-start-build-test`.
+
+
```

Comment:
This section introduces a link to build Omega, but it does not explicitly state that building the develop branch and the test branch of Omega are required steps for the subsequent instructions even though those information is provided in following sections. Adding one or two statements explaining the Omega build requirements and how they relate to the `DEVELOP_BUILD` and `TEST_BRANCH_BUILD` paths would be helpful for users trying this for the first time.